packages/syft/src/syft/core/node/enums.py

```diff
@@ -37,3 +40,8 @@ class ResponseObjectEnum(str, Enum):
     USER = "user"
     WORKER = "worker"
     DATA = "data"
+
+
+class TensorPointerStatus(str, Enum):
+    READY = bcolors.green("Ready")
+    PROCESSING = bcolors.yellow("Processing")
```

Comment:
Could also add them to packages/syft/core/tensor.py --> `TensorPointer`

Reply:
Solved by: [8164bae](https://github.com/OpenMined/PySyft/pull/7105/commits/8164bae204debcfdeaf15d377c61bf68f7f5ef2b)

Reply:
TensorPointer doesn't implement `__repr__` making him use his parent (_Pointer_) __repr__ method. 
<img width="663" alt="tensor_pointer_ready" src="https://user-images.githubusercontent.com/26658472/203660181-06a41a2f-b2a0-4668-8c69-476377489fee.png">